doc/source/howtos/upgrade_guide.rst

```diff
@@ -57,6 +57,16 @@ Before you start the upgrade, move your custom settings from
 Ubuntu) or from :file:`/etc/percona-xtradb-cluster.conf.d/wsrep.cnf` (on Red Hat
 and CentOS) to the new location accordingly.
 
+.. rubric:: ``caching_sha2_password`` Is the Default Authentication Plugin
+
+In |PXC| |version|, the default authentication plugin is
+``caching_sha2_password``. HAProxy (see :ref:`haproxy`) does not support this
```

Comment:
Should this refer to ProxySQL (instead of HAProxy)?

Reply:
Removed this reference to HAProxy.